### PR TITLE
Revert "Versioned integration context - add 5.5 build"

### PR DESCRIPTION
**Pull request by Itay4**
this is no longer needed

### CODE REVIEW

File: Packs/Base/pack_metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "Base",
     "description": "The base pack for Cortex XSOAR.",
     "support": "xsoar",
-    "currentVersion": "1.2.1",
+    "currentVersion": "1.2.0",
```
Comment (glicht) [inline Packs/Base/pack_metadata.json:5]: Version needs to bump forward.

File: Packs/Base/ReleaseNotes/1_2_1.md [hunk 2]
```diff
@@ -1,4 +0,0 @@
-
```
Comment (glicht) [inline Packs/Base/ReleaseNotes/1_2_1.md:1]: Don't delete this file.

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 3]
```diff
@@ -3233,23 +3233,18 @@ def __call__(self):
 get_demisto_version = GetDemistoVersion()
 
 
-def is_demisto_version_ge(version, build_number=''):
+def is_demisto_version_ge(version):
```
Comment (glicht) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:3236]: I would leave this change

**Comment by Itay4** [comment]
ill open a new pull request with just modifying `is_versioned_context_available`

**Pull request closed without merging**